lib/components/App.js

```diff
@@ -5,36 +5,48 @@ const remote = electron.remote;
 const ipc = electron.ipcRenderer
 const mainProcess = remote.require('./main.js');
 import ImageCard from './ImageCard.jsx';
+// import { SketchField, Tools } from 'react-sketch';
+import ReactPaint from 'react-paint';
 
 export default class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      imgData: ''
+      img: ''
     }
-    this.screenshot = this.screenshot.bind(this)
-    this.imgData = this.imgData.bind(this)
   }
 
   componentWillMount() {
-    ipc.on('imageList', (event, content) => {
-      this.setState({ imgData: content })
+    ipc.on('currentImg', (event, img) => {
+      console.log('img ', img);
+      this.setState({ img })
     })
   }
 
-  screenshot() {
-    mainProcess.enableScreenshot()
-  }
-
   render() {
+    console.log(this.state.img);
+    const props = {
+      style: {
+        background: 'tomato',
+      },
+      brushCol: '#ffffff',
+      lineWidth: 10,
+      className: 'react-paint',
+      height: 500,
+      width: 500
+    };
+
+
     return (
-      console.log(this.state.imgData)
       <div>
         <h1>Hey Gurl!</h1>
-        <button onClick={this.screenshot}
-                className='btn'>
-                screenshot
-        </button>
+        {/* <canvas width='200px' height='200px'></canvas> */}
+        {/* <SketchField height='300px'
+                     width='200px'
+                     tool={Tools.Pencil}
+                     color='black'
+                     lineWidth={3}/> */}
+        <ReactPaint {...props} />
```

Comment:
@ejwill04 I'm guessing this is from the react-sketch module, not the react-paint one